.github/ISSUE_TEMPLATE/investigation_or_research.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Investigation
+about: 
+title: ' Investigate/ Research on ...'
+labels: 'T-investigation'
+assignees: ''
+---
+
+## Summary
+<!-- A clear and concise description of what should be investigated or researched is about. -->
+
+## Related resources
+<!-- List of additional resource that might be usefull when doing research or investigation -->
+
+## Acceptance criteria
+<!-- Acceptance criteria in terms of investigation or research usually (but not only) should consist of:
+- list of open questions that need to be answered 
+- some summary, research paper or meta knowledge as an outcome of research or investigation
+- at least two people sign off. One of them is the Tech lead second should be decided with team
+- workshop or knowledge sharing session should be conducted to share outcomes
+-->
+[] add open questions here..
+[] Summary, research paper or meta knowledge written and merged to main repo
```

Comment:
I'd say this could also sometimes be an issue comment. An example would be the sync worker pool issue. The question would be "Can we optimize full sync by increasing concurrency?" and the answer is in the issue itself. I don't see how and where it would make sense to put this in a file in the repo.

Reply:
same as above

Reply:
added (if applicable) statement